packages/app/src/components/Sliders/Slider.tsx

```diff
@@ -0,0 +1,50 @@
+import ui from './Slider.module.css'
```

Comment:
Why do we need both a `RangeEditor` and `Slider`?

Reply:
We don't, didn't yet refactor this part, but the idea would be to use Slider underneath range editor